app/controllers/sessions_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
     user      = User.find_by_auth_hash(auth_hash) || User.new
 
     user.assign_from_auth_hash(auth_hash)
+    user.sync_notifications
```

Comment:
If we're doing it in the controller we can remove the after create callback here: https://github.com/octobox/octobox/blob/master/app/models/user.rb#L10